editor/js/libs/ui.three.js

```diff
@@ -149,20 +178,19 @@ class UITexture extends UISpan {
 		if ( texture !== null ) {
 
 			const image = texture.image;
+			canvas.title = texture.sourceFile;
+			const scale = canvas.width / image.width;
```

Comment:
It is not safe to access `image.width` at this point. This must be done inside the next `if` block.